ray-on-gke/kuberay-tpu-webhook/main.go

```diff
@@ -300,6 +300,28 @@ func mutateRayCluster(admissionReview *admissionv1.AdmissionReview) (*admissionv
 	return admissionResponse, nil
 }
 
+func hasWorkerID(container corev1.Container) bool {
+	if container.Env != nil && len(container.Env) > 0 {
+		for _, envVar := range container.Env  {
+			if envVar.Name == "TPU_WORKER_ID" {
+				return true
+			}
+		}
+	}
+	return false
+}
+
+func hasWorkerName(container corev1.Container) bool {
+	if container.Env != nil && len(container.Env) > 0 {
+		for _, envVar := range container.Env {
+			if envVar.Name == "TPU_WORKER_NAME" {
```

Comment:
I thought it was TPU_WORKER_HOSTNAMES?

Reply:
This was just a typo I missed, it needed to be "TPU_NAME".